boards/PDM/Inc/App/App_LowVoltageBatteries.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <stdbool.h>
+
+/**
+ * Allocate and initialize low voltage batteries
+ * @note The low voltage batteries are modeled after the 18650s on our vehicle,
+ *       which are connected to:
+ *         - A cell balancing IC
+ *         - A charging IC
+ *         - A boost controller IC
+ * @param is_overvoltage A function that can be called to check if the cell
+ *                       balancing IC for the low voltage batteries senses an
+ *                       overvoltage
+ * @param has_charge_fault A function that can be called to check if the cell
+ *                         balanacing IC for the low voltage batteries has a
+ *                         fault
+ * @param has_boost_fault A function that can be called to check if the boost
+ *                        controller IC for the low voltage batterise has a
+ *                        fault
+ * @return The created low voltage batteries, whose ownership is given to the
+ *         caller
+ */
+struct LowVoltageBatteries *App_LowVoltageBatteries_Create(
```

Comment:
`LowVoltageBatteries` represent the two 18650 cells on the car

Reply:
Just as a general thought, should these be plural if we're effectively looking at them as a single battery?

Reply:
I was considering this too, let's change it!